### ISSUE TITLE
Automated ML training tool

### ISSUE THREAD

**Issue by lorikrammer** [GW-HIVE/PredictMod#312]
Develop an automated ML training tool for which users would only need to upload a dataset and the tool will train and compare several different models.

**Comment by lorikrammer** [comment]
@pmcneely Will this tool assist in generating BCOs or point users to the BCO portal?

**Comment by pmcneely** [comment]
@lorikrammer At least in the first iteration it will only be focused on the data -> AI/ML model pipeline. We can certainly point folks to the BCO portal. We will need to discuss if we want a more complex setup after the initial release.

**Comment by lorikrammer** [comment]
That sounds good to me, I just wanted to mention the connection to BCOs in case we'd like to explore that down the road.

**Comment by lorikrammer** [comment]
@pmcneely do you plan to make any adjustments to the tool prior to publication? If so, I'll keep this ticket open. Otherwise, let's close it and open new tickets for feature improvements going forward. Thanks!

**Comment by pmcneely** [comment]
@lorikrammer Yes, the tool really needs some attention this month. I will put together a few things that I think are both achievable and needed for use, and prep the blurb (#317) to reflect the combined updates by the end of January.

**Comment by pmcneely** [comment]
Initial release is ready. BCO implementation will need to be moved to another ticket.
